template/deployments/helm/charts/templates/_helpers.tpl

```diff
@@ -1,7 +1,7 @@
 {{/*
```

Comment:
I think we should remove this altogether and move the resulting definitions into the `Values.yaml` files

Reply:
Helm does not support complex template logic directly within the values.yaml file. The values file should contain simple key-value pairs. So, moving the complex templating logic back to the _helpers.tpl file and reference those helpers in the templates.